.storybook/main.js

```diff
@@ -5,6 +5,7 @@ const config = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
+    'storybook-addon-sass-postcss',
```

Comment:
#5233 will add this, so maybe should keep a eye on that if that gets merged first 👀